GameController.js

```diff
@@ -1,91 +1,115 @@
-class GameController {
-
-	createEnemy() {
-		snakeLiving = assetManager.getSprite("assets");
-		snakeLiving.regX = 50;
-		snakeLiving.regY = 50;
-		snakeLiving.x = enemyXPos;
-		snakeLiving.y = enemyYPos;
-		snakeLiving.gotoAndPlay("snakeAlive");
-		stage.addChild(snakeLiving);
-		console.log("loading");
+function GameController() { 
```

Comment:
This was refactored to be more inline with what I do when I make constructor objects. Your other approach was fine too, but you have 3 unique ways of declaring objects you're using in this project alone across three files. Try to find your preferred approach and stick with one. Consistency is key in JS land.